src/showmime.jl

```diff
@@ -42,6 +45,9 @@ function Base.show(io::IO, mime::MIME"image/png", img::AbstractMatrix{C};
     end
 end
 
+Base.show(io::IO, mime::MIME"image/png", img::OffsetArray{C}; kwargs...) where C =
```

Comment:
should `C` be restricted to `Colorant`? I guess it doesn't matter for a non `Colorant` when the `MethodError(show, ...)` is thrown one level deeper (it should just continue to walk the display stack). But I am guessing that it messes with the result of `showable` which falls back to `hasmethod`. thoughts?